### ISSUE TITLE
parallel hang in itaps test

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#2360]
Eric reports that sometimes, itaps test hangs in parallel. but, scalable parallel works alright.

I ran valgrind on the tests and see a slew of memory errors.

The itaps libs were updated recently and perhaps memory errors are causing problems for parllel execution.

Long story short, some cases in itaps.py are skipped in parallel by adding logic to the .py file. This is because just adding tests to skip list does NOT prevent actual execution (only diffing logic) and so skip list cannot be used to avoid the hang.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. As such, not all 
information was able to be captured in the transition. Below is
a complete record of the original redmine ticket.

Ticket number: 340
Status: Pending
Project: VisIt
Tracker: Bug
Priority: Normal
Subject: parallel hang in itaps test
Assigned to: 
Category: 
Target version: 
Author: Mark Miller
Start: 08/10/2010
Due date: 
% Done: 0
Estimated time: 
Created: 08/10/2010 11:06 pm
Updated: 08/18/2010 05:18 pm
Likelihood: 3 - Occasional
Severity: 2 - Minor Irritation
Found in version: 2.0.1
Impact: 
Expected Use: 
OS: All
Support Group: Any
Description:
 Eric reports that sometimes, itaps test hangs in parallel. but, scalable parallel works alright.

I ran valgrind on the tests and see a slew of memory errors.

The itaps libs were updated recently and perhaps memory errors are causing problems for parllel execution.

Long story short, some cases in itaps.py are skipped in parallel by adding logic to the .py file. This is because just adding tests to skip list does NOT prevent actual execution (only diffing logic) and so skip list cannot be used to avoid the hang.

Comments:
